Console/Command/GuestOrder.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace TNW\Stripe\Console\Command;
+
+use Magento\Framework\App\Area;
+use Magento\Framework\App\State;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use TNW\Stripe\Model\Order\Guest;
+/**
+ * Class GuestOrder
+ * @package TNW\Stripe\Console\Command
```

Comment:
package directive is redundant - see magento 2 coding standard or use their codesniffer to check

Reply:
thanks - removed

Reply:
Thanks - removed